src/components/TextInput/TextInput.jsx

```diff
@@ -1,96 +1,77 @@
 // @flow
-import { toUpper } from 'lodash';
-import React, { type Node } from 'react';
+import React, { Component, type Node } from 'react';
 import cx from 'classnames';
+
 import { getDataAttrs } from '../../dataUtils';
 import type { Data } from '../../types';
+
 import styles from './style.scss';
 
 type Props = {|
-  placeholder: string,
-  value: string,
-  onChange: () => void,
-  onFocus: () => void,
-  onBlur: () => void,
-  suffix: Node,
+  data?: Data,
+  disabled?: boolean,
+  error?: boolean,
+  maxLength?: number,
+  name?: string,
+  onChange?: (SyntheticKeyboardEvent<HTMLInputElement>) => void,
+  onClick?: (SyntheticMouseEvent<HTMLInputElement>) => void,
+  onFocus?: (SyntheticFocusEvent<HTMLInputElement>) => void,
+  onBlur?: (SyntheticFocusEvent<HTMLInputElement>) => void,
+  placeholder?: string,
   prefix: Node,
-  isDisabled: boolean,
-  disabled: boolean,
-  size: number,
-  error: boolean,
-  maxLength: string,
-  data: Data,
   id?: string,
-  name?: string,
   style?: { [key: string]: any },
+  suffix: Node,
+  type: 'text' | 'password' | 'email' | 'url',
+  value?: string,
 |};
 
-const TextInput = ({
-  suffix,
-  prefix,
-  placeholder,
-  value,
-  onChange,
-  onFocus,
-  onBlur,
-  isDisabled,
-  disabled,
-  size,
-  error,
-  data,
-  maxLength,
-  id,
-  name,
-  style,
-}: Props) => {
-  const fixStyle = cx(styles.fix, {
-    [styles.suffix]: suffix,
-    [styles.prefix]: prefix,
-  });
+class TextInput extends Component<Props> {
+  static defaultProps = {
+    placeholder: 'Type something...',
+  };
 
-  const wrapperStyle = cx(styles.wrapper, {
-    [styles.wrapperDisabled]: isDisabled || disabled,
-    [styles.wrapperError]: error,
-    [styles.wrapperLarge]: toUpper(size) === 'LARGE',
```

Comment:
Since this was removed, it is a breaking change. This would need to be added back or addressed before we move to this version of Plasma.

Reply:
This will be addressed, and hopefully caught early by Flow where possible.

The design system doesn't address this style at all, from what I saw.

Reply:
Since the property is removed, when this is merged into Plasma. Right after, the version would need to updated and any dependent repositories moved to this version and fixed (to account for this change). 

Otherwise, we would have a merged breaking change that would go unnoticed until someone else updated the Plasma version.

Reply:
Since the property is removed, right after this is merged into Plasma, the version would need to updated and any dependent repositories moved to this version and fixed (to account for this change). 

Otherwise, we would have a merged breaking change that would go unnoticed until someone else updated the Plasma version.